R/call_derivation.R

```diff
@@ -5,6 +5,12 @@
 #'
 #' @param dataset The input dataset
 #' @param derivation The derivation function to call
+#'
+#'   A function is expected. The function must provide the `dataset` argument
+#'   and all arguments specified in the `params()` objects passed to the
+#'   `variable_params` and `...` argument. Therefore it is not possible to
+#'   specify dplyr functions like `mutate()` or `summarize()`.
+#'
```

Comment:
```suggestion
#'
#'   A function is expected. The function must provide the `dataset` argument
#'   and all arguments specified in the `params()` objects passed to the
#'   `variable_params` and `...` argument. Therefore, it is not possible to
#'   specify `{dplyr}` functions like `mutate()` or `summarize()`.
#'
```

Reply:
Maybe Therefore is the wrong word??  Perhaps *Please Note*

Reply:
Updated